posts/2023-07-24-leadership-council-update.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "July 2023 Leadership Council Update"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Hello again from the Rust Leadership Council. In our [first blog post][fr1st post], we laid out several immediate goals for the council and promised to report back on their progress. It has been about a month since our first update so we wanted to share how it's going and what we're working on now.
```

Comment:
nit: "fr1st post" => "f1rst post" or just "first post" - if we change this don't forget to change the usage at the end of the section.